src/01.UI-Elements/UndoButton.md

```diff
@@ -0,0 +1,3 @@
+```js padded
+<UndoButton visibility="shown"/>
```

Comment:
visibility? Where is that declared? Couldn´t find that.

Reply:
Good point. Thanks! I forgot to change it here.